R/tm_g_lineplot.R

```diff
@@ -294,6 +302,21 @@ tm_g_lineplot <- function(label,
                           pre_output = NULL,
                           post_output = NULL,
                           ggplot2_args = teal.widgets::ggplot2_args()) {
+  if (lifecycle::is_present(strata)) {
+    warning(
+      "The `strata` argument of `tm_g_lineplot()` is deprecated as of teal.modules.clinical 0.9.1. ",
+      "Please use the `group_var` argument instead.",
+      call. = FALSE
+    )
+    group_var <- strata
+  }
```

Comment:
Not really sure why it does this sometimes, but this is the fix I've uesed elsewhere for it
```suggestion
  } else {
    strata <- group_var # resolves missing argument error
  }
```